src/aggregators/is-performant/index.js

```diff
@@ -20,7 +20,7 @@
 const Aggregate = require('../aggregate');
 
 /** @type {string} */
-const firstMeaningfulPaint = require('../../audits/performance/first-meaningful-paint').name;
+const firstContentfulPaint = require('../../audits/performance/first-contentful-paint').name;
```

Comment:
Also, I'm about to put up a PR for real fMP... so can you bring the filename back to fmp?  :)